vm/actor/Cargo.toml

```diff
@@ -20,8 +20,7 @@ cid = { package = "forest_cid", version = "0.3", features = ["cbor"] }
 serde = { version = "1.0", features = ["derive"] }
 lazy_static = "1.4.0"
 ipld_blockstore = "0.1"
-ipld_hamt = { version = "1.0", features = ["go-interop"] }
-ipld_amt = { features = ["go-interop"], version = "0.2" }
+ipld_hamt = { path = "../../ipld/hamt" }
```

Comment:
why can't these be removed (hamt and amt below)

Reply:
??? I removed the old ones from crates and added the local onces

Reply:
oh oops thought I was reading a diff cargo toml

Reply:
haha i had a feeling :p